www/src/css/snaplist.css

```diff
@@ -34,6 +44,10 @@
   margin-top:140%;
 }
 
+.b-store .b-snaplist_grid .b-snaplist__item:before {
```

Comment:
This applies to whole `b-snaplist-grid`.
Is `b-snaplist__item` ever used outside the grid? It seems like using just `.b-snaplist__item` without nesting would be enough as it's not used outside of the grid anywhere.

If different styles are needed for `b-store` they ideally should be added as modifier to base class not nested under `b-store`.